src/helpers/response_message.coffee

```diff
@@ -0,0 +1,69 @@
+class ResponseMessage
+
+  teamCreated: (team)->
+    "#{team.label()} created, add some people to it"
+
+  teamAlreadyExists: (teamName)->
+    "#{teamName} team already exists"
```

Comment:
Why not pass `team` and use `team.label()` here and in `teamDeleted` and `teamNotFound`? to make it consistent